airbyte-cdk/python/pyproject.toml

```diff
@@ -35,6 +35,7 @@ isodate = "~0.6.1"
 Jinja2 = "~3.1.2"
 jsonref = "~0.2"
 jsonschema = "~3.2.0"
+pandas = "2.2.0"
```

Comment:
This change causes the smoke tests on shopify and zendesk-support not to run. This is unfortunate. @alafanechere @clnoll is there a way to have the --use-local-cdk flag and make sure all the dependencies are resolved? [This](https://github.com/airbytehq/airbyte/blob/04f40f903c1e901748d7dcdbb753188346a6e2a9/.github/workflows/cdk_connectors_tests.yml#L95) is the command that is failing right now

Reply:
The error could indeed indicate that the connector is not built with the local CDK.
The logic to install the local CDK is handled [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/dagger/actions/python/common.py#L240).
I checked the dagger logs of your run and it looks like its correctly called.
There could be a clash between where the local CDK is installed via pip and where the connector is installed via poetry (we're not using venv on build so I'd not expect this)...
Now that the CDK is on poetry I think it'd be better to install the local CDK via `poetry` instead of `pip`.
TLDR; we should investigate if the local cdk is really installed when the `--use-local-cdk` flag is passed.

Reply:
[GH issue](https://github.com/airbytehq/airbyte-internal-issues/issues/8494). CC @natikgadzhi this is preventing us from testing CDK changes on connectors

Reply:
Sounds like a high priority regression testing bug to me. I'll spend a week looking into them myself when I'm back, and will see if I can snatch a week of Augustin's time too.

Reply:
You can ignore this. This was the issue: https://github.com/airbytehq/airbyte/pull/45362